TheWarWithin/Priorities/RogueAssassination.simc

```diff
@@ -1,13 +1,13 @@
 ## https://github.com/simulationcraft/simc/blob/thewarwithin/ActionPriorityLists/default/rogue_assassination.simc
-## SimulationCraft Commit Sync: 96a5d2e
-## Date: 2025-08-12
+## SimulationCraft Commit Sync: f7fc80f
+## Date: 2025-08-26
 
 actions.precombat+=/apply_poison
 # Check which trinket slots have Stat Values
 actions.precombat+=/variable,name=trinket_sync_slot,value=1,if=trinket.1.has_use_buff&(!trinket.2.has_use_buff|trinket.1.cooldown.duration>=trinket.2.cooldown.duration)&!trinket.2.is.treacherous_transmitter|trinket.1.is.treacherous_transmitter|trinket.1.is.house_of_cards
 actions.precombat+=/variable,name=trinket_sync_slot,value=2,if=trinket.2.has_use_buff&(!trinket.1.has_use_buff|trinket.2.cooldown.duration>trinket.1.cooldown.duration)&!trinket.1.is.treacherous_transmitter|trinket.2.is.treacherous_transmitter|trinket.2.is.house_of_cards
 # Determine combo point finish condition
-actions.precombat+=/variable,name=effective_spend_cp,value=cp_max_spend-2<?5*talent.hand_of_fate
+actions.precombat+=/variable,name=effective_spend_cp,value=cp_max_spend-2<?5
```

Comment:
Does this calculate as expected?  It may need to be `(cp_max_spend-2)<?5`.  Right now, it may effectively be `combo_points.max - max( 2, 5 )` and result in